docs/content/en/docs-dev/user-guide/managing-piped/required-permissions.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
[imo] How about moving to the installation section? It might be nice for users to know the requirements when installing.
https://pipecd.dev/docs-v0.47.x/installation/install-piped/

Reply:
Sounds great, thank you!

Reply:
i fixed